### PR TITLE
Hide/Unhide themes documented

### PR DESCRIPTION
**Pull request by escopecz**
Change that documents https://github.com/mautic/mautic/pull/13723

### CODE REVIEW

**Comment by escopecz** [comment]
I'm not sure how to deal with the last 2 Vale issues. Any suggestions?

File: docs/themes/manage_themes.rst [hunk 1]
```diff
@@ -104,6 +104,31 @@ To preview a Theme:
...
+
+2. Click the **Settings** icon.
+
+3. Click **Themes**.
+
+4. In the Themes section, locate the Theme that you want to hide.
+
+5. Select the drop-down before the Theme, and click **Hide**.
+
+6. Mautic moves the Theme to the bottom of the table and show it in grey. This means the Theme is hidden in the builders.
```
Comment (RCheesley) [inline docs/themes/manage_themes.rst]:
```suggestion
6. Mautic moves the Theme to the bottom of the table and show it in grey, which hides it within builders.
```

Vale fix

File: docs/themes/manage_themes.rst [hunk 2]
```diff
@@ -104,6 +104,31 @@ To preview a Theme:
...
+
+To revert this change open the context menu of the hidden Theme and click **Unhide**.
```
Comment (RCheesley) [inline docs/themes/manage_themes.rst:129]: This should really be 'reveal' or something like that - unhide isn't really a great term to use. Is this something we have in Core already @escopecz ? If so we can wrap this line in a vale off/on statement.

**Review by RCheesley — CHANGES_REQUESTED** [review]
One fix, and one query about the terminology used.

File: docs/themes/manage_themes.rst (hunk 2 above)
Comment (escopecz) [reply docs/themes/manage_themes.rst:129]: Yes, it's in the core already

File: docs/themes/manage_themes.rst [hunk 3]
```diff
@@ -104,6 +104,31 @@ To preview a Theme:
...
+2. Click the **Settings** icon.
+
+3. Click **Themes**.
+
+4. In the Themes section, locate the Theme that you want to hide.
+
+5. Select the drop-down before the Theme, and click **Hide**.
+
+6. Mautic moves the Theme to the bottom of the table and show it in grey, which hides it within builders.
+
```
Comment (RCheesley) [inline docs/themes/manage_themes.rst:128]:
```suggestion
.. vale off

```

File: docs/themes/manage_themes.rst [hunk 4]
```diff
@@ -104,6 +104,31 @@ To preview a Theme:
...
+To revert this change open the context menu of the hidden Theme and click **Unhide**.
+
```
Comment (RCheesley) [inline docs/themes/manage_themes.rst:131]:
```suggestion
.. vale on

```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Let's wrap that bit in a Vale Off so it doesn't flag the error. I'll make a PR to suggest a change to the wording.

The vale on/off needs a line break above and below, it wasn't totally clear from the GitHub UI whether that works with my suggestions so please just make sure the spacing comes out alright! :)

**Comment by RCheesley** [comment]
https://github.com/mautic/mautic/pull/13835 PR to improve the grammar here :)

File: docs/themes/manage_themes.rst (hunk 2 above)
Comment (RCheesley) [reply docs/themes/manage_themes.rst:129]: Yep, made the PR to address that :)

**Pull request merged**